packages/main/src/MessageView.ts

```diff
@@ -0,0 +1,246 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import SegmentedButton, { SegmentedButtonSelectionChangeEventDetail } from "./SegmentedButton.js";
```

Comment:
We have agreement to add type keyword after import keyword when import is used only as a type but there is limitation in type script so please do following changes: 
```ts
import  type { SegmentedButtonSelectionChangeEventDetail } from "./SegmentedButton.js";
import SegmentedButton from "./SegmentedButton.js";
```

Reply:
The proposed change causes the eslint to fail because of duplicate imports from the "./SegmentedButton.js"